plugins/request/src/main/java/org/apache/unomi/plugins/request/actions/SetRemoteHostInfoAction.java

```diff
@@ -232,7 +232,7 @@ private boolean ipLookupInDatabase(String remoteAddr, Session session) {
             }
             return true;
         } catch (IOException | GeoIp2Exception e) {
-            logger.debug("Cannot resolve IP", e);
+            logger.warn("Cannot resolve IP", e);
```

Comment:
use debug to log the stack trace and only display  the error message as a warning